packages/core/src/external/fhir/adt-encounters.ts

```diff
@@ -158,6 +184,51 @@ export async function getAdtSourcedEncounter({
   return JSON.parse(fileData.toString());
 }
 
+/**
+ * If an existing encounter exists, merge the new bundle into it
```

Comment:
"If an existing encounter exists..." -> "If we already have an encounter that represents ADT messages..."

Just because it's a clearer on the ADT aspect of the Encounter and it helps to explain what ADT sourced means.

Reply:
awesome, done (i think?)